src/DSSTest.sol

```diff
@@ -20,6 +20,7 @@ import "forge-std/Test.sol";
 import {GodMode} from "./GodMode.sol";
 import {MCD,Ilk,MCDMainnet,MCDGoerli} from "./MCD.sol";
 import {MCDUser} from "./MCDUser.sol";
+import {RootDomain} from "./domains/RootDomain.sol";
```

Comment:
sorry for the early approval, thought this was done, but based on your suggestion in our chat, this should change to `HostDomain`

Reply:
ok, then I got confused about what you were saying in the chat.  If we are using Host/Guest on the bridge, where would that apply here? only if we had L2<>L3 and we would call the L2 the Host?

Reply:
`Host`/`Guest` applies to the OptimismDomain and ArbitrumDomain to indicate which is the rollup and which is the parent chain.

For the test suite we need to anchor this on some chain, so I'd call it the root. Root will usually be either Ethereum or Goerli as it is the L1. You can think of it like Root = L1 and the Host/Guest refers to relative pairings for the bridge. Does that clear it up?